### PR TITLE
Fix help/referenceDocs/etc ribbon buttons not opening

### PR DESCRIPTION
**Pull request by Zlatkovsky**

### CODE REVIEW

File: packages/common/src/safe.external.urls.ts [hunk 1]
```diff
@@ -2,14 +2,16 @@ export default {
   playground_help: 'https://github.com/OfficeDev/script-lab/blob/master/README.md',
   ask: 'https://stackoverflow.com/questions/tagged/office-js',
   excel_api:
-    'https://dev.office.com/reference/add-ins/excel/excel-add-ins-reference-overview',
+    'https://docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/excel-add-ins-reference-overview',
```
Comment (lexahall) [inline packages/common/src/safe.external.urls.ts]: Is there somewhere that we document these URLs?

File: packages/editor/src/pages/AddinCommands/setup.ts [hunk 2]
```diff
@@ -2,6 +2,11 @@ import { Utilities, HostType } from '@microsoft/office-js-helpers';
 import safeExternalUrls from 'common/lib/safe.external.urls';
 
 export default function setup() {
+  // SUPER IMPORTANT NOTE:  The add-in commands code doesn't do a redirect to localhost
+  //   (or whatever other environment).  So it always run in production.
```
Comment (lexahall) [inline packages/editor/src/pages/AddinCommands/setup.ts]: NIT: always runs

File: packages/editor/src/pages/External/index.tsx [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
-  };
+  // Note: using just an indexOf of whatever follows "#/external-page?destination="
+  //    rather than doping an actual search string query.
+  //    This is because, as part of launching the dialog, Office.js prepends a bunch
+  //    of stuff on the URL as a query string (?_host_Info=) BEFORE the hash,
+  //    and doesn't seem to want a URL with an existing query string in there.
+  //    On the routing side, in turn, it appears that having the destination
+  //    embedded directly after the "#/external-page/<encoded url>"
+  //    isn't working either.  So just end up with two "?" on the URL,
+  //    which -- though weird -- seems to work just fine.
```
Comment (lexahall) [inline packages/editor/src/pages/External/index.tsx]: NIT: this comment is a little confusing. I read it a few times before I really got what you were saying. Maybe try summarizing a bit?

File: packages/common/src/safe.external.urls.ts (hunk 1 above)
Comment (Zlatkovsky) [reply packages/common/src/safe.external.urls.ts]: I got them from Alex.  The old ones were no longer functional.

**Comment by Zlatkovsky** [comment]
Fixes #734
Fixes #780

File: packages/editor/src/pages/External/index.tsx (hunk 3 above)
Comment (Zlatkovsky) [reply packages/editor/src/pages/External/index.tsx]: Ok, shortened it.  It's a really strange issue, but I think in practice as long as it works, no one should need to touch it again...

**Pull request merged**